api/envoy/extensions/load_balancing_policies/pick_first/v3/pick_first.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package envoy.extensions.load_balancing_policies.pick_first.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.load_balancing_policies.pick_first.v3";
+option java_outer_classname = "PickFirstProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/load_balancing_policies/pick_first/v3;pick_firstv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Pick First Load Balancing Policy]
+// [#extension: envoy.load_balancing_policies.pick_first]
+
+// This configuration allows the built-in PICK_FIRST LB policy to be configured via the LB policy
+// extension point. See the :ref:`load balancing architecture overview
+// <arch_overview_load_balancing_types>` for more information.
```

Comment:
Is this going to get implemented in Envoy? Do you have these docs ready? Are you going to submit the code in the final PR? I don't have any context on this, thank you.

/wait-any

Reply:
No, I don't expect this to be implemented in Envoy. We do have a `pick_first` LB policy in gRPC and we want the ability to configure this policy as the leaf policy via the custom LB policy mechanism in xDS. I do have a [document](https://github.com/grpc/proposal/pull/357) out for the changes we are planning to make to the `pick_first` LB policy which covers this.

@markdroth: Do you want to add anything?

Reply:
OK then please remove all the docs and just add the not-implemented-hide annotation. Then you can remove most of the other changes. Thank you.

/wait

Reply:
Just to confirm, this is a policy we have in gRPC, but this particular policy is unlikely to be implemented in Envoy.

Adding the not-implemented-hide annotation and removing the Envoy documentation sounds good.

Reply:
Done. Thanks.

Reply:
Can you please remove the arch_voerview stuff also? It's not real. Just have the not-implemented-hide annotation and perhaps a short note that this is currently used in gRPC only. Thanks.

/wait

Reply:
Done. Thanks.